test-results/user-journeys-Poker-—-Quiz-1ce88-er-click-changes-page-state-chromium/error-context.md

```
@@ -0,0 +1,225 @@
+- Following Playwright test failed.
+- Explain why, be concise, respect Playwright best practices.
+- Provide a snippet of code with the fix, if possible.
+
+# Test info
+
+- Name: user-journeys.spec.ts >> Poker — Quiz Deep Interaction >> quiz answer click changes page state
+- Location: e2e/user-journeys.spec.ts:363:2
+
+# Error details
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)